### ISSUE TITLE
New Rule: Empty Args

### ISSUE THREAD

**Issue by Snugug** [sasstools/sass-lint#38]
With Sass, if a function or mixin doesn't have any arguments, the parenthesis can be left off of the definition and, in the case of mixins, the invocation as well

**Comment by bgriffith** [comment]
Started to look into this one. Experience is slightly different. Parenthesis can be left off of the definition of mixins but not functions. When I try I get a Sass error;
`@function baz ": expected "(", was "{"`

While I can drop the parenthesis off both the invocation of functions and mixins.

```scss
// Causes an error
@function baz {
  @return 'baz';
}

// Is ok
.baz {
  content: baz;
}

// Is ok
@mixin qux {
  content: 'baz';
}

// Is ok
.qux {
  @include qux;
}
```

**Comment by Snugug** [comment]
Check out this Sass Meister:
http://sassmeister.com/gist/17649e207a82c4efb946

The function isn't being invoked if it doesn't include `()`, you just happen to have named the function and its return the same, making it appear to be OK.

There is, however, a discrepancy between Libsass and Ruby Sass as to whether a function can be defined w/o `()`. Libsass allows it, Ruby Sass does not.

**Comment by bgriffith** [comment]
Haha oh dear. Amateur mistake!

**Comment by Snugug** [comment]
I trusted you man! I _trusted you_! :stuck_out_tongue_closed_eyes:

**Comment by bgriffith** [comment]
Hahaha :grin: - Thoughts on above?

**Comment by Snugug** [comment]
Yah, only for mixins makes sense to me
